File: ExerciciosC/nome_n_vezes.cpp
#include<conio.h>
#include<stdio.h>

main(){
	
	char nome[30];
	int numero = 0;
	
	printf("Digite um nome = ");
	scanf("%s", &nome);
	
	printf("\nDigite o numero de vezes que deseja repiti-lo = ");
	scanf("%d", &numero);
	
	for(int i = 1; i <= numero; i++){
		printf("\n%s", nome);
	}
	
}

File: ExerciciosJava/NomeNVezes.java
import java.util.Scanner;

public class NomeNVezes {

	public static void main(String[] args) {
		
		int numero;
		String nome;
		
		Scanner s = new Scanner(System.in);
		System.out.println("Digite seu nome");
		nome = s.nextLine();
		System.out.println("Digite o numero de vezes que queira repetir seu nome = ");
		numero = s.nextInt();
		
		for(int i = 1; i <= numero; i++) {
			System.out.println(nome);
		}
		
	}

}
